### PR TITLE
	Fix month completion with two digits

### LINKED ISSUE #23: cp $HOME/lib/cli-template.sh my-new-script
my-new-script

### PR DESCRIPTION
**Pull request by StudioEtrange**

### CODE REVIEW

**Comment by aks** [comment]
Thanks for the bug fix. 

Please update an existing test or write a new test to confirm your change.  

The date tests are in `test-date-utils.sh`.

**Review by aks — APPROVED** [review]

**Comment by aks** [comment]
I reviewed the function being "corrected" more carefully, and realized that the change was incorrect.  `df_month` is _supposed_ to have leading blanks on 1 digit month numbers.  See `df_month0` for leading-zero month digits.

**Comment by StudioEtrange** [comment]
Yes.
df_month0 code is similar to the content of this PR.
6 years ago in june 2016, when this patch was made but not merged, df_month0 did not exists yet
df_month0 was introduced in september 2016 in this commit https://github.com/aks/bash-lib/commit/d7c007f737586cd89a9c4c915220c94347b6f259#diff-46bb7810c1bf31f20b814e280a62db7b944aabd7d174e5bd5734c2c77125a54eR865

File: date-utils.sh [hunk 1]
```diff
@@ -678,7 +678,7 @@ df_day() {		        # d - the day of the month as a decimal number (01-31).
 }
 
 df_month() {		        # e - day of the month as a decimal number (1-31); 
-  printf "%2d\n" $month         #     single digits are preceded by a blank.
```
Comment (dh-orko) [inline date-utils.sh:681]: - printf "%02d\n" $month  #

**Review by dh-orko — APPROVED** [review]

**Pull request closed without merging**